src/python/WMCore/Services/WMArchiver/DataMap.py

```diff
@@ -247,6 +252,8 @@ def createArchiverDoc(job_id, fwjr, version=None):
     newfwjr['meta_data'] = {'agent_ver': version,
                          'host': socket.gethostname().lower(),
                          'fwjr_id': job_id,
-                         'ts': int(time.time())
+                         'jobtype': jobtype,
```

Comment:
@ticoann , Seangchan does it means we need to tweak WMArchive schema again and add jobtype and jobstate to it, see https://github.com/dmwm/WMArchive/blob/master/test/data/samples/fwjr_production.py#L4